blocks/select.py

```diff
@@ -177,6 +177,12 @@ def recursion(brick):
             for child in brick.children:
                 for path, param in recursion(child).items():
                     new_path = Path([Path.BrickName(brick.name)]) + path
+                    if new_path in result:
+                        raise ValueError(
+                            "name collision while retrieving parameters (" +
```

Comment:
Can you please split this error message into a concise one and a verbose one like we do it [here](https://github.com/bartvm/blocks/blob/master/blocks/main_loop.py#L181)?